osf/management/commands/manage_switch_flags.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+import logging
+
+from django.core.management.base import BaseCommand
+
+from osf.features import switches, flags
+from waffle.models import Flag, Switch
+
+logger = logging.getLogger(__name__)
+
+def manage_waffle():
+    file_switches = [switches[switch] for switch in switches]
+    current_switches = Switch.objects.values_list('name', flat=True)
+
+    add_switches = set(file_switches) - set(current_switches)
+    for switch in add_switches:
+        Switch.objects.get_or_create(name=switch, defaults={'active': False})
+        logger.info('Adding switch: {}'.format(switch))
+
+    delete_switches = set(current_switches) - set(file_switches)
+    Switch.objects.filter(name__in=delete_switches).delete()
```

Comment:
The delete functionality is a little dangerous. I wonder if we should make that an optional function, and by default only add things. If we are a bit slow in releases vs. front end, we might end up removing a flag or switch that is currently being used because it was added manually while waiting for us.

Reply:
I'll add a command line argument to this. Running the script from the admin app will not delete flags or switches.